.github/workflows/pipeline.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       - name: Install cosign
         # Third-party action, pin to commit SHA!
         # See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions
-        uses: sigstore/cosign-installer@7cc35d7fdbe70d4278a0c96779081e6fac665f88
+        uses: sigstore/cosign-installer@c3667d99424e7e6047999fb6246c0da843953c65
```

Comment:
Where does the new commit SHA come from?

Reply:
The cosign installer now installs version 2.0 (not experimental). In [their blogpost](https://blog.sigstore.dev/cosign-2-0-released/) they describe some of the breaking changes. I've added the flag to say yes to the prompts. Running the build image workflow on this branch seems to work. I would push it to production.